nym-registration-client/src/builder/config.rs

```diff
@@ -170,14 +181,26 @@ fn mixnet_debug_config(mixnet_client_config: &MixnetClientConfig) -> DebugConfig
 
     debug_config.cover_traffic.disable_loop_cover_traffic_stream =
         mixnet_client_config.disable_background_cover_traffic;
+    if let Some(poisson_rate) = mixnet_client_config.poisson_rate {
+            let duration = std::time::Duration::from_millis((poisson_rate as f64).round() as u64);
+        debug_config.cover_traffic.loop_cover_traffic_average_delay = duration;
 
+    }
     if let Some(min_mixnode_performance) = mixnet_client_config.min_mixnode_performance {
         debug_config.topology.minimum_mixnode_performance = min_mixnode_performance;
     }
 
     if let Some(min_gateway_performance) = mixnet_client_config.min_gateway_performance {
         debug_config.topology.minimum_gateway_performance = min_gateway_performance;
     }
+    if let Some(avg_packet_ms) = mixnet_client_config.average_packet_delay {
+        debug_config.traffic.average_packet_delay = Duration::from_millis(avg_packet_ms as u64);
+    debug_config.acknowledgements.average_ack_delay=Duration::from_millis(avg_packet_ms as u64);
+    }
+
+    if let Some(msg_delay_ms) = mixnet_client_config.message_sending_average_delay {
+        debug_config.traffic.message_sending_average_delay = Duration::from_millis(msg_delay_ms as u64);
+    }
     log_mixnet_client_config(&debug_config);
```

Comment:
Please add those new options to `log_mixnet_client_config` so we can see what's happening?